roles/cis/tasks/3/3.5.1.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: "3.5.1 Ensure DCCP is disabled (Not Scored)"
+  vars:
+   modprobe_conf: /etc/modprobe.d/modprobe.conf
+  block:
+    - name: "register the stat output of modprobe.d directory"
+      stat:
+        path: /etc/modprobe.d
+      register: output_1
+    - name: "create modprobe.d directory if it does not exist"
+      file:
+        path: /etc/modprobe.d
+        state: directory
+      when: output_1.stat.exists == false
+    - name: "register the stat output of modprobe.conf"
+      stat:
+        path: /etc/modprobe.d/modprobe.conf
+      register: output_2
+    - name: "create modprobe.conf if it does not exist"
+      file: path=/etc/modprobe.d/modprobe.conf state=touch owner=root group=sys mode=0555
```

Comment:
Should the file mode be 0644 instead of 0555? Already existing files in /etc/modprobe.d/ folder has 0644 file permissions.

Reply:
Fixed!